keystore/src/connection/platform/wasm/storage.rs

```diff
@@ -116,7 +190,7 @@ impl WasmEncryptedStorage {
                 }
             }
             WasmStorageWrapper::InMemory(map) => {
-                if let Some(store) = map.get(collection) {
```

Comment:
Since we are touching the piece of code anyway, maybe we wanna get rid of this deep nesting which makes it hard to understand and use this more linear version instead? If you agree that it's a better style

```rust
WasmStorageWrapper::InMemory(map) => {
                let Some(store) = map.borrow().get(collection) else {
                    return Ok(None);
                };
                let Some(js_value) = store.get(id.as_ref()).cloned() else {
                    return Ok(None);
                };
                let Some(mut entity) = serde_wasm_bindgen::from_value::<Option<R>>(js_value)? else {
                    return Ok(None);
                };
                entity.decrypt(&self.cipher)?;
                Ok(Some(entity))
            }
 ```

Reply:
Fixed in: https://github.com/wireapp/core-crypto/pull/695/commits/a5d0af601d22f843c8706b15d3641fba14c72229